public/app/features/alerting/unified/Analytics.ts

```diff
@@ -214,6 +215,16 @@ export const trackInsightsFeedback = async (props: { useful: boolean; panel: str
   reportInteraction('grafana_alerting_insights', { ...defaults, ...props });
 };
 
+interface RuleVersionComparisonProps {
+  latest: boolean;
+  oldVersion: number;
+  newVersion: number;
+}
+
+export const trackRuleVersionsComparisonClick = async (payload: RuleVersionComparisonProps) => {
+  reportInteraction('grafana_alerting_rule_versions_comparison_click', { ...payload });
+};
+
```

Comment:
Can give us an idea of how broad a range of versions are being compared

Reply:
Could we use fullstory instead of this custom reporting?

Reply:
I imagine we might be able to? But I was thinking of us being able to see how far back a user is going, e.g. are they comparing version `10` and `11`? Or version `1` and `479`?

Is there an easy way we can calculate that with FullStory?

Reply:
Ah, interesting. Such thing can be tricky with fullstory. I didn't pay enought attention to the payload object